kafka_consumer/datadog_checks/kafka_consumer/client.py

```diff
@@ -228,9 +228,13 @@ def _get_consumer_groups(self):
             consumer_groups_future = self.kafka_client.list_consumer_groups()
             try:
                 list_consumer_groups_result = consumer_groups_future.result()
+                for valid_consumer_group in list_consumer_groups_result.valid:
+                    self.log.debug("Discovered consumer group: %s", valid_consumer_group.group_id)
 
                 consumer_groups.extend(
-                    valid_consumer_group.group_id for valid_consumer_group in list_consumer_groups_result.valid
+                    valid_consumer_group.group_id
+                    for valid_consumer_group in list_consumer_groups_result.valid
+                    if valid_consumer_group.group_id != ""
```

Comment:
Could it be (for whatever reason) `None`? In such case you could just 

```suggestion
                    if valid_consumer_group.group_id
```